src/CommunityToolkit.Maui/AppBuilderExtensions.shared.cs

```diff
@@ -34,7 +40,10 @@ public static MauiAppBuilder UseMauiCommunityToolkit(this MauiAppBuilder builder
 		});
 
 		Popup.RemapForControls();
+
+#if ANDROID
 		NavigationBar.RemapForControls();
+#endif
```

Comment:
This removes the warning "This call site is reachable on: 'MacCatalyst' 15.0 and later. 'NavigationBar.RemapForControls()' is only supported on: 'Android' 21.0 and later"